src/JSInterop/Microsoft.JSInterop.JS/src/src/Microsoft.JSInterop.ts

```diff
@@ -450,7 +450,7 @@ export module DotNet {
 
   function formatError(error: Error | string): string {
       if (error instanceof Error) {
-          return `${error.message}\n${error.stack}`;
+          return error.stack ?? error.message;
```

Comment:
I noticed that this exception formatting includes the exception message twice, since `error.stack` already contains the exception message. Hence this change, which uses only `error.stack`, or `error.message` if `stack` is falsey.

Reply:
If I remember correctly this could be different on different browsers. 
FF vs chrome have different syntax.

Reply:
Ah, you're right. I've just reverted this change.